src/standalone/userJupyterServer/userServerUrlProvider.ts

```diff
@@ -306,16 +321,17 @@ export class UserJupyterServerUrlProvider implements IExtensionSyncActivationSer
             const baseUrl = `${url.protocol}//${url.host}${url.pathname === '/lab' ? '' : url.pathname}`;
 
             const token = `${url.searchParams.get('token')}`;
```

Comment:
we were unnecessarily storing the text `null` and passing that around, instead of using an `empty string` or `undefined`